lab0/INSTRUCTIONS.md

```diff
@@ -70,17 +72,15 @@ nr1 nerdpack:serve
 
 ![lab0-launcher](../screenshots/lab0_screen01.png) <sup>Note: The icon might be a little different from the screenshot.</sup>
```

Comment:
![Lab 0 launcher](../screenshots/lab0_screen01.png)
![Your launcher on the apps page](../screenshots/lab0_screen01.png)